### PR TITLE
Readds headpatting and slapping but not shittily

### PR DESCRIPTION
**Pull request by Cyberboss**
Reverts tgstation/tgstation#28028

Reee I'll fix it

### CODE REVIEW

**Comment by Expletive** [comment]
Now you're even more my favorite

**Comment by ohnopigeons** [comment]
24 hours

**Comment by lzimann** [comment]
Kneejerk revert.

**Comment by YakumoChen** [comment]
Closing a change you merged a revert of, merging an unpopular PR, unclose it

**Comment by VeryRealAccount** [comment]
Izimann confirmed for hating fun.

**Comment by optimumtact** [comment]
it wasn't that unpopular, 11 - 16

**Comment by YakumoChen** [comment]
33% more downvotes then upvotes seems unpopular to me

**Comment by PKPenguin321** [comment]
Closing a revert to a PR you merged...

**Comment by DoctoreMad** [comment]
>Not closing reverts to PRs you merge

Come on its not like you don't even /tg/ coderbus hell
In other news when do we bring over our current host as a maintainer

**Comment by KorPhaeron** [comment]
Reopening and immediatley merging a PR that two other maintainers closed seems like a violation of the spirit of the 24 hour rule.

**Comment by optimumtact** [comment]
you should be in irc when you're maintaining kor

**Comment by Chocobro** [comment]
i swear to god if i see an "emergency revert" for this

**Comment by Iamgoofball** [comment]
@KorPhaeron 

if they want it back they can re-add it complying with codebase standards with Datum Emotes instead of hardcodinng it into intents, an already messy piece of garbage.

**Comment by Iamgoofball** [comment]
also get on IRC or steam memelord

**Comment by Iamgoofball** [comment]
@PKPenguin321 Kneejerk reverts are also against the rules.

**Comment by DoctoreMad** [comment]
@Iamgoofball Funny you say that its not supposed to be merged in first place
Not that it matters because the whole arguing about the rules is because of the dangerous **NIPPON BANZAI** energies and coders are trying to hold it back by removing or nerfing features

By the way NIPPON BANZAI

**Comment by Iamgoofball** [comment]
whine_more

**Comment by lzimann** [comment]
If this is added in this state, I will remove again.

**Comment by DoctoreMad** [comment]
Not an argument

**Comment by lzimann** [comment]
It is actually an argument.

File: code/modules/mob/living/carbon/human/species.dm [hunk 1]
```diff
@@ -1082,7 +1082,18 @@
...
+	var/aim_for_mouth  = user.zone_selected == "mouth"
+	var/target_on_help_and_unarmed = target.a_intent == INTENT_HELP && !target.get_active_held_item()
+	var/target_aiming_for_mouth = target.zone_selected == "mouth"
+	var/target_restrained = target.restrained()
+	if(aim_for_mouth && ( target_on_help_and_unarmed || target_restrained || target_aiming_for_mouth))
+		playsound(target.loc, 'sound/weapons/slap.ogg', 50, 1, -1)
+		user.visible_message("<span class='danger'>[user] slaps [target] in the face!</span>",
+			"<span class='notice'>You slap [target] in the face! </span>",\
+		"You hear a slap.")
+		target.endTailWag()
```
Comment (Iamgoofball) [inline code/modules/mob/living/carbon/human/species.dm]: here's proof this wasn't added for the purpose of bitchslapping people

this was added as fetish fuel

Comment (Lezus-Konrad) [inline code/modules/mob/living/carbon/human/species.dm]: you shitting me?

**Comment by Grazyn** [comment]
Confusedrock added it, and he hates lizards with a passion, he's the uncle ruckus of lizardkin

File: code/modules/mob/living/carbon/human/species.dm (hunk 1 above)
Comment (Grazyn) [inline code/modules/mob/living/carbon/human/species.dm]: Confusedrock added it, and he hates lizards with a passion, he's the uncle ruckus of lizardkin

Comment (PKPenguin321) [inline code/modules/mob/living/carbon/human/species.dm]: >Confusedrock added it, and he hates lizards with a passion

Good one

Comment (KorPhaeron) [inline code/modules/mob/living/carbon/human/species.dm]: Grazyn is correct

Comment (optimumtact) [inline code/modules/mob/living/carbon/human/species.dm]: yeah it was added for that reason only

**Comment by lzimann** [comment]
@Cyberboss

**Comment by PopNotes** [comment]
this is some pathetic shit

**Comment by Cyberboss** [comment]
meh

**Comment by Cyberboss** [comment]
Someone else can do this

**Pull request closed without merging**